woof-code/rootfs-skeleton/usr/share/doc/HOWTO_sns.htm

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
```

Comment:
I recommend:
1. Move this file to `woof-code/rootfs-packages/simple_network_setup/usr/share/doc/HOWTO_network-setup.html` (or a more creative name)
2. Move `HOWTO_connman-gtk` to `woof-code/rootfs-petbuilds/connman-gtk/usr/share/doc/HOWTO_network-setup.html`
3. Change the link in `HOWTO_Internet.htm` so it points to `HOWTO_network-setup.html`, and make the text more generic ("Network setup wizard" or something). This way, it will point to documentation for connman-gtk in a Puppy with connman-gtk, or to documentation for SNS in a Puppy with SNS.

Otherwise, users might get confused because they need to know which tool is used by *this* Puppy and look at the relevant documentation, and some users will probably look for a second network setup tool in the menu if they don't get along with the first one they tried.

Reply:
I'll do that. See https://forum.puppylinux.com/viewtopic.php?p=73641#p73641 and tell if that would be the correct way to do it.

Reply:
And yes, I wanted to ask if these two are the only ones which are used in all puppies? And if removing `HOWTO_Internet.htm` will be safe or not?

Reply:
I got the answer for the first one. Now doing the requested changes.

Reply:
@dimkr, few doubts making this. Should I move them to `.../usr/share/doc` or `.../usr/share/doc/sns`.

And will these files be automatically copied, or will I need to manually setup `pinstall.sh` to do that?

Reply:
> And yes, I wanted to ask if these two are the only ones which are used in all puppies?

AFAIK yes, I *think* somebody used NetworkManager but it's probably a modified woof-CE.

Reply:
> @dimkr, few doubts making this. Should I move them to `.../usr/share/doc` or `.../usr/share/doc/sns`.

Doesn't matter really, although I think a subdirectory would be nice for extra files like images or extra .html files, beyond the one index.html points to.

> And will these files be automatically copied, or will I need to manually setup `pinstall.sh` to do that?

Automatically.

Reply:
Thanks! I'll do it.

Reply:
@dimkr, will `3builddistro` still need the changes done after I complete this?

Reply:
@dimkr, there is no SNS petbuild dir! What to do about that?

Reply:
If I understand correctly, yes. It tries to delete /usr/share/doc and needs to skip the Puppy documentation, regardless of where it comes from in the woof-CE source tree.